front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/DisplayItinerary/Destination.tsx

```diff
@@ -62,6 +63,7 @@ function Destination(props: DestinationProps) {
               className="btn btn-sm btn-only-icon btn-white ml-auto"
               type="button"
               onClick={() => {
+                dispatch(clearVias());
```

Comment:
Why clear vias ? I don't think so, we should be able to change destination without clear all the settings of the path !